shell/initialize/client.js

```diff
@@ -589,6 +617,8 @@ function checkForErrors(app) {
 
 // When navigating on a different route but the same component is used, Vue.js
 // Will not update the instance data, so we have to update $data ourselves
+// Why are we modifying this behaviour? If component is rendered, then we deal with it... It is what it is...
```

Comment:
There is something strange with vue, which is alluded to in the comment above. if a component is re-used then something about it's content remains. they talk about changing route, i thought i saw it on the same page. So this bit is them working around that.